Analyse_Birds.qmd

```diff
@@ -12,57 +12,419 @@ output:
    toc_float: true
 ---
 
-Le lien du site : https://zenodo.org/records/8251495 
 
 
-Pistes 
+# Data presentation
 
-Calcul 
+The dataset (available following this link  https://zenodo.org/records/8251495 ) used for this work contains nest records from different studies conducted in North America and more specifically in Wisconsin and Illinois. It was conducted in 13 different counties : Dane, Grant, Green, Iowa, Lafayette, Monroe, Rock, Lee, Ogle, Will, Grundy, Carroll, and Jo Daviess counties. The dataset contains the records of 3257 nests and for each of them different observations.
 
-Taux de succès 
-Taux de survie
+These observations are the following:
```

Comment:
Maybe explain that these are the variables in the dataset and that you are presenting them and their respective levels.

Reply:
I will change this and be more specific